zipline/testing/fixtures.py

```diff
@@ -480,8 +548,10 @@ class WithBcolzDailyBarReader(WithTmpDir, WithSimParams):
     """
     BCOLZ_DAILY_BAR_PATH = 'daily_equity_pricing.bcolz'
     BCOLZ_DAILY_BAR_LOOKBACK_DAYS = 0
-    BCOLZ_DAILY_BAR_FROM_CSVS = False
     BCOLZ_DAILY_BAR_USE_FULL_CALENDAR = False
+    BCOLZ_DAILY_BAR_START_DATE = alias('START_DATE')
+    BCOLZ_DAILY_BAR_END_DATE = alias('END_DATE')
+    _write_method_name = 'write'
```

Comment:
I think the bigger indication that this is not the supported way to modify this is that it is private. I can add a quick comment explaining why this exists though.